docs/docs/faq.md

```diff
@@ -5,57 +5,19 @@
 Nebari regenerates this file on every run. This means it will be removed by the operating system during its cleanup process,
 but running the `nebari deploy` command again as a Nebari administrator will update/create a `NEBARI_KUBECONFIG` file for you.
 
-## How are Nebari conda user environments created? Who creates them?
+## How are conda user environments created? Who creates them?
 
-The short answer: there are currently _two_ ways of creating environments, as we are in the process of migrating Nebari to conda-store,
-and so which way depends on your use-case.
+`Conda-store` manages all environments on the Nebari. It allows users to create private environments in their own namespace, or shared environments under a group namespace.
 
-The longer answer:
+Additionally, there is a legacy approach which is still available in Nebari. Administrators can create global environments by specifying the environment in `nebari-config.yml`. Environments specified in this way will be made available for all users and services under the `nebari-git` namespace on `conda-store`. We do not recommend this approach as it requires a redeploy in order to update the environment, though it may be useful in some circumstances.
```

Comment:
It can be nice to have some environments available for your initial Nebari deployment.  The build is still done by conda store.  This method works, is tested, and we have no other way to do this.  I think we should keep this functionality in Nebari and just say this is an alternative way to create envs.